Week2/homework/trigger_countrylanguages.sql

```diff
@@ -0,0 +1,11 @@
+CREATE DEFINER=`hyfuser`@`localhost` TRIGGER `countrylanguage_AFTER_INSERT` AFTER INSERT ON `countrylanguage` FOR EACH ROW BEGIN
```

Comment:
your trigger is correct, but it would have been better to define it BEFORE insert and not AFTER so mysql has less work to do

Reply:
yes now I got the point.
On the other hand: the way how the question is written makes the idee not clear (for me). 
The question was: If a country X has 9 languages in the CountryLanguage table, and a user INSERTs one more row in the CountryLanguage table, then I should get an alert.